server/monitor/subscriptions.go

```diff
@@ -155,28 +184,42 @@ func (m *Monitor) checkGlobalChatsSubscription(msteamsSubscriptionsMap map[strin
 
 	// Check if all chats subscription is not present on MS Teams
 	if _, msteamsSubscriptionFound := msteamsSubscriptionsMap[mmSubscription.SubscriptionID]; !msteamsSubscriptionFound {
+		m.api.LogInfo("Creating global chats subscription")
+
 		// Create all chats subscription on MS Teams
-		m.CreateAndSaveChatSubscription(mmSubscription)
+		m.createAndSaveChatSubscription(mmSubscription)
 		return
 	}
 
 	if mmSubscription.Certificate != m.certificate {
+		m.api.LogInfo("Recreating global chats subscription on certificate mismatch")
+
 		if err := m.recreateGlobalSubscription(mmSubscription.SubscriptionID, mmSubscription.Secret); err != nil {
 			m.api.LogError("Unable to recreate all chats subscription", "error", err.Error())
 		}
 	}
 
-	if time.Until(mmSubscription.ExpiresOn) < (5 * time.Minute) {
+	if shouldRefresh(mmSubscription.ExpiresOn) {
+		if isExpired(mmSubscription.ExpiresOn) {
+			// In the future, this won't need to be an error if we can resync, but for
+			// now notify a human.
+			m.api.LogError("Global chats subscription expired")
```

Comment:
Can we provide some context or next steps for that human?

Reply:
I like this idea, though would propose we host that externally, e.g. "what to do on ERROR log <...>". The number of actual error logs is now quite manageable, thankfully.